wasm-rpc/src/lib.rs

```diff
@@ -48,13 +48,17 @@ pub mod wasmtime;
 use crate::builder::WitValueBuilder;
 pub use builder::{NodeBuilder, WitValueBuilderExtensions};
 pub use extractor::{WitNodePointer, WitValueExtractor};
+use std::borrow::Cow;
+use std::ops::Deref;
 
 #[cfg(not(feature = "host"))]
 #[cfg(feature = "stub")]
 pub use bindings::golem::rpc::types::{NodeIndex, RpcError, Uri, WasmRpc, WitNode, WitValue};
 
 #[cfg(feature = "host")]
 use ::wasmtime::component::bindgen;
+use golem_wasm_ast::analysis::{AnalysedResourceId, AnalysedResourceMode};
+use wasm_wave::wasm::{WasmType, WasmValue, WasmValueError};
```

Comment:
Don't let your IDE do these imports, it breaks the feature flags. Either import under a feature condition, or use fully qualified names.

Reply:
Done (I think) - will not resolve conversation until you verify. Thanks